src/block/Anvil.php

```diff
@@ -104,8 +104,10 @@ public function tickFalling() : ?Block{
 	public function onHitGround(FallingBlock $blockEntity) : void{
 		foreach($this->position->getWorld()->getCollidingEntities($blockEntity->getBoundingBox()) as $ent){
 			if($ent instanceof Living){
-				$damageSource = new EntityDamageByEntityEvent($blockEntity, $ent, EntityDamageEvent::CAUSE_FALLING_BLOCK, min($blockEntity->fallDistance * 2, 40));
-				$ent->attack($damageSource);
+				if($blockEntity->fallDistance > 1) {
+					$damageSource = new EntityDamageByEntityEvent($blockEntity, $ent, EntityDamageEvent::CAUSE_FALLING_BLOCK, min($blockEntity->fallDistance * 2, 40));
```

Comment:
Should the damage be casted to int? Or is passing floats valid / correct in the case of falling blocks?

Reply:
Does vanilla implement the damage as integers? I assumed they also do floating point damage.